### PR TITLE
Cache LFS Objects in the TARDIS-Carsus Compatibility Check Workflow

### PR DESCRIPTION
**Pull request by atharva-2001**
### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

**Comment by epassaro** [comment]
Could you rebase this PR to master?

File: .github/workflows/bridge.yml [hunk 1]
```diff
@@ -91,8 +91,32 @@ jobs:
...
+      - name: Create LFS file list
+        run: git lfs ls-files -l | cut -d' ' -f1 | sort > .lfs-assets-id
+        working-directory: tardis-refdata
+
+      - name: Restore LFS cache
+        uses: actions/cache@v2
+        id: lfs-cache
+        with:
+          path: tardis-refdata/.git/lfs
+          key: ${{ runner.os }}-lfs-${{ hashFiles('tardis-refdata/.lfs-assets-id') }}-v1
```
[the remark this answers is not in the document]
Comment (epassaro) [reply .github/workflows/bridge.yml:109]: we use a `CACHE_NUMBER: 0` variable on top of the file for that (I think setting to 1 is better though) and then replace `v1` with `v${{ env.CACHE_NUMBER }}`

[the remark this answers is not in the document]
Comment (chvogl) [reply .github/workflows/bridge.yml:109]: I agree with @epassaro.

**Review by chvogl — APPROVED** [review]
Looks good to me.

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**